charts/sonarqube/values.yaml

```diff
@@ -258,6 +258,21 @@ prometheusExporter:
   securityContext:
     runAsUser: 1000
     runAsGroup: 1000
+  
+  # Prometheus Pod Monitor
+  # ref: https://github.com/coreos/prometheus-operator
+  #
+  podMonitor:
+    # Create PodMonitor Resource for Prometheus scraping
+    enabled: true
```

Comment:
podMonitor should be disabled by default, because it requires prometheus-operator installed.
The same for sonarqube-dce/lts

Reply:
Hi,

the podMonitor template is checks if prometheus and podmonitor is enabeld, so we never deploy the monitor without prometheus. 

```
{{- if and .Values.prometheusExporter.enabled .Values.prometheusExporter.podMonitor.enabled }}
```

I can change the default if you want :)

Reply:
Ah got your point, yes i change the default! sorry

Reply:
fixed :)